playbooks/appsemblerPlaybooks/update_custom_requirements.yml

```diff
@@ -0,0 +1,115 @@
+- hosts: "edxapp-server"
+  become: true
+
+  # Update extra requirements, private requirements, 
+  # collect static assets, and restart edx app and workers
+
+  tasks:
+    - include_vars: roles/common_vars/defaults/main.yml
+    - include_vars: roles/edxapp/defaults/main.yml
+
+
+    - name: install read-only ssh key
+      copy: >
+        content="{{ EDXAPP_GIT_IDENTITY }}" dest={{ edxapp_git_identity }}
+        force=yes owner={{ edxapp_user }} mode=0600
+      when: EDXAPP_USE_GIT_IDENTITY
+
+    - name: set git fetch.prune to ignore deleted remote refs
+      shell: git config --global fetch.prune true
+      sudo_user: "{{ edxapp_user }}"
+
+    # this is meant to create a new private requirements file, not
+    # the base in edx-platform/requirements/edx-private.txt
+    # if you are running this playbook you should be overriding 
+    # private_requirements_file
+    - name: Create private requirements file
+      template:
+        src: ../roles/edxapp/templates/private_requirements.txt.j2
```

Comment:
This relative directory seems fragile. Is it possible there's a better way of handling this?

Reply:
Relative paths like this are used in a lot of places in the configuration repo.